.gitlab-ci/unit-tests.yml

```diff
@@ -0,0 +1,41 @@
+---
+unit-tests:
+  extends: .job
+  stage: unit-tests
+  variables:
+    SHELLCHECK_VERSION: v0.6.0
+    ANSIBLE_INVENTORY: inventory/local-tests.cfg
+    ANSIBLE_REMOTE_USER: root
+    ANSIBLE_BECOME: "true"
+    ANSIBLE_BECOME_USER: root
+    ANSIBLE_VERBOSITY: "3"
+  script:
+    - /usr/bin/python -m pip install -r tests/requirements.txt
```

Comment:
Shell inlined in YAML file is a pain. Can we move this into one or several .sh file?